consensus/service/src/config/tokens.rs

```diff
@@ -5,12 +5,69 @@
 use crate::consensus_service::ConsensusServiceError;
 use mc_common::HashSet;
 use mc_consensus_enclave::FeeMap;
+use mc_crypto_keys::{DistinguishedEncoding, Ed25519Public};
+use mc_crypto_multisig::SignerSet;
 use mc_transaction_core::{tokens::Mob, Token, TokenId};
-use serde::{Deserialize, Serialize};
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
 use std::{fs, iter::FromIterator, path::Path};
 
+/// A helper struct for ser/derserializing a SignerSet that uses hex-encoded
```

Comment:
I just assumed that hex-encoded DER bytes are a good way to represent an Ed25519 key in the configuration file. But maybe the raw bytes are just as good? Any thoughts on that @jcape?